specutils/tests/test_analysis.py

```diff
@@ -483,7 +483,11 @@ def test_fwhm():
 
     spectrum = Spectrum1D(spectral_axis=wavelengths, flux=flux)
     result = fwhm(spectrum)
-    assert result == 0.9090909090909092*u.um
+    # Note that this makes a little more sense than the previous version;
+    # since the maximum value occurs at wavelength=1, and the half-value of
+    # flux (0.5) occurs at exactly wavelength=2, the result should be
+    # exactly 1 (2 - 1).
+    assert result == 1.0*u.um
```

Comment:
How about modifying that test (or adding another one) so that wavelength=2 is not actually on the grid, to test the interpolation? E.g.
`wavelengths = np.linspace(1, 10, 31) * u.um`

Reply:
@dhomeier can you open another issue to address expanding on the unit testing? I'd like to merge this PR now for a release and iterate on it over time.

Reply:
Sure. Just wanted to note that the PR adds some functionality for which there is no test coverage now, but if it has time I can look into it sometime.

Reply:
@dhomeier, I've gone ahead and included a test for this interpolation case. No need to make a new issue.